frontend/src/app/organization/organization-list/organization-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { OrganizationService } from '../organization.service';
+import { Organization } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-list',
+  templateUrl: './organization-list.component.html',
+  providers: [OrganizationService]
+})
+export class OrganizationListComponent {
+
+  organizations: Organization[];
+
+  constructor(private orgService: OrganizationService) {
+    this.orgService.query().subscribe((orgs: Organization[]) => this.organizations = orgs)
```

Comment:
That's really an ugly pattern - you don't have to manually subscribe.
Don't subscribe, make observable public, use `async` pipe in the template.